### DISCUSSION TITLE
The production build is not working.

### DISCUSSION THREAD

**Discussion by gitnupur** [webpack/webpack#17886] [Q&A]
# Bug report

**What is the current behavior?**
1: The code is working in development mode always, but in production build in container_entry.js promiseall fails. So it is not able to find the modules. This never fails in development mode. 

2: On clicking of tab that should load another MFE, whenever the start.js in webpack gets it, the app loads succesfully. 

var moduleMap = {
	"./Page1": function() {
		return Promise.all([__webpack_require__.e(469), __webpack_require__.e(560), __webpack_require__.e(529), __webpack_require__.e(103), __webpack_require__.e(381), __webpack_require__.e(725), __webpack_require__.e(47), __webpack_require__.e(592), __webpack_require__.e(335), __webpack_require__.e(633), __webpack_require__.e(472), __webpack_require__.e(763), __webpack_require__.e(310), __webpack_require__.e(913), __webpack_require__.e(595), __webpack_require__.e(392)]).then(function() { return function() { return (__webpack_require__(42950)); }; });
	},
	"./Page2": function() {
		return Promise.all([__webpack_require__.e(560), __webpack_require__.e(529), __webpack_require__.e(103), __webpack_require__.e(381), __webpack_require__.e(107), __webpack_require__.e(592), __webpack_require__.e(633), __webpack_require__.e(472), __webpack_require__.e(763), __webpack_require__.e(310), __webpack_require__.e(913), __webpack_require__.e(846), __webpack_require__.e(273)]).then(function() { return function() { return (__webpack_require__(22537)); }; });
	}
};

This piece is not called. 

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
The Whenever Tab change happens , the above code should work. 

**Other relevant information:**
webpack version:  "webpack": "^5.73.0",
Node.js version: v18.17.0
Operating System: Windows 10
Additional tools: React 18.2.0  , "typescript": "^4.7.4",

**Comment by alexander-akait** [comment]
Please create reproducible test repo, thank you
